contracts/modules/cover/Cover.sol

```diff
@@ -423,7 +436,10 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon {
       PoolAllocation storage allocation = allocations[i];
       IStakingPool _stakingPool = stakingPool(allocation.poolId);
 
-      uint nxmBurned = allocation.coverAmountInNXM * burnAmount / segment.amount;
+      uint nxmBurned = allocation.coverAmountInNXM
+        * burnAmount / segment.amount
+        * GLOBAL_CAPACITY_DENOMINATOR / globalCapacityRatio;
+
       _stakingPool.burnStake(cover.productId, segment.start, segment.period, nxmBurned);
 
       allocation.coverAmountInNXM -= SafeUintCast.toUint96(nxmBurned);
```

Comment:
I think there's something fishy here. Assuming a 2x global capacity ratio and a cover worth 10 NXM. If a payout on a claim is 10 NXM, the coverAmountInNXM should be 0 aftewards, not 5 NXM. I believe this partial burn only makes sense in the context of staking when calling burnStake.